.goreleaser.yaml

```diff
@@ -0,0 +1,94 @@
+before:
+  hooks:
+    - go mod tidy
+    - go mod vendor
+    - go mod download
+    - go test -v ./...
+    #- go generate ./...
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm
+      - arm64
+      - "386"
```

Comment:
Really... 386?

Reply:
It's on the Makefile, I assume they want to support it, so included it, just like they have arm6 and arm7